Robitaille/template/README.md

```diff
@@ -11,3 +11,8 @@ and this other part does something.  I forget why that I did it:
     [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]
 
 If you can't use it, its kind of your problem, not mine!
+
+There is a functon for finding prime Numbers:
```

Comment:
I suggest using more consistent capitalization in your sentences.  Feel free to consult a style guide or ask a friendly collaborator. Like @eteq.  Hi friendly British person.  I like you.